tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -92,6 +105,56 @@ In Rider:
 - Set **MSBuild version** to **.NET Core**.
 - Install the **Godot support** plugin.
 
+Create a New Solution in Rider
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Start by creating a new solution in Rider. Navigate to the appropriate option and select it.
+
+   .. image:: img/rider_new_solution.webp
+      :alt: New Solution in Rider
+
+2. Choose the "Class Library" option from the list of templates.
+
+   .. image:: img/rider_choose_class_library.webp
+      :alt: Choose Class Library
+
+3. You can optionally select the "Put solution and project in the same directory" checkbox for your convenience.
+
+   .. image:: img/rider_solution_and_project_same_dir.webp
+      :alt: Solution and Project in Same Directory
+
+Setting Up Godot with Rider Solution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. In Godot, create a new project in the same directory you used for Rider. Ignore the warning about not empty path.
+
+   .. image:: img/rider_godot_new_project.webp
+      :alt: New Project in Godot
+
+2. Make sure to set the Version Control Metadata to "None", because Rider has initialized it already.
+
+Testing and Troubleshooting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Test your setup with a simple "Hello World" class. Remember to attach the script to a Node for it to run properly.
+
+   .. image:: img/rider_test_hello_world.webp
+      :alt: Test Hello World
+
+2. If you encounter a Mono error in Godot, such as:
+
+   ``Duplicate 'global::System.Runtime.Versioning.TargetFrameworkAttribute'``
+
+   This issue can be resolved by going back to Rider and excluding the `obj` folder from the project.
```

Comment:
Perhaps, I wasn't the first person to have such an issue so I've mentioned everything what prevented me from building the project

Reply:
It's because the original solution Rider created wasn't using `Godot.NET.Sdk`, so it was likely using default `BaseOutputPath` and `OutputPath`, got built (hence the `obj/` folder), and then I imagine we have a process to correct the `.csproj` so it works with Godot? In any case, it arises from creating a wrong solution at the beginning of the process.